tests/test_callback.py

```diff
@@ -12,6 +12,7 @@ def run_playbook_callback(tmpdir, report_type):
     extra_env = {}
     ansible_version = get_ansible_version()
     if ansible_version is None:
+        print("Ansible version is None!")
```

Comment:
Shouldn't the next line have told you the same?

Reply:
A leftover.